Makefile

```diff
@@ -320,9 +320,11 @@ define run_smartredis_tests_with_standalone_server
 	echo "Running standalone tests" && \
 	PYTHONFAULTHANDLER=1 python -m pytest $(SR_TEST_PYTEST_FLAGS) $(COV_FLAGS) \
 		$(SKIP_DOCKER) $(SKIP_PYTHON) $(SKIP_FORTRAN) \
-		--build $(SR_BUILD) --sr_fortran $(SR_FORTRAN) $(1) && \
+		--build $(SR_BUILD) --sr_fortran $(SR_FORTRAN) $(1)  ; \
+	(testresult=$$?; \
 	echo "Shutting down standalone Redis server" && \
 	python utils/launch_redis.py --port $(SR_TEST_PORT) --nodes 1 --stop && \
+	exit $$testresult) && \
```

Comment:
I think this would be slightly more bash-like:
```
test $$testresult -eq 0 || echo "Tests failed"; exit $$test_result
```

Reply:
I like that, will adopt